src/style.css

```diff
@@ -1,3 +1,35 @@
-body {
-  margin: 0;
+.body {
+  font-family: Roboto, sans-serif;
+}
+
+nav {
+  font-size: 18px;
+  margin-top: 10px;
+  margin-right: 10px;
+  text-align: center;
+
+}
+
+.list {
+  margin-left: 20px;
+  margin-top: 10px;
+  color: black;
+  text-align: top;
+  line-height: 40px;
+}
```

Comment:
Be consistent with your vertical margins. You have `margin-top: 10px;` for the `.list` class. It's recommended to add only top or only bottom margin, but not both. Decide which one you want to use and stick with it throughout your CSS.

Reply:
fixed